tests/integration/modifiers/scroll-into-view-test.js

```diff
@@ -45,23 +45,34 @@ module('Integration | Modifier | scroll-into-view', function (hooks) {
 
   test('it renders when shouldScroll resolves to true', async function (assert) {
     this.options = { test: true };
-    this.shouldScroll = Promise.resolve(true);
+    this.resolve;
```

Comment:
I think this line is extraneous and can be removed

Reply:
Or, instead of putting it on `this`, just create a variable in scope with `let`.

```
let resolvePromise;
this.shouldScroll = new Promise((resolve) => (resolvePromise = resolve));

// ...

await resolvePromise(true);
```

Reply:
Changed